crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -16,7 +16,7 @@
 
 //! Primitives for Subspace consensus.
 
-#![deny(warnings)]
+#![feature(let_else)]
```

Comment:
Honestly, I'm more inclined to not introduce more nightly features, even though we now have to install the nightly for compiling the runtime to wasm anyway. Ideally, it's nice that we could just compile the binary under stable.